src/event/log_event.rs

```diff
@@ -170,15 +171,6 @@ impl IntoIterator for LogEvent {
     }
 }
 
-impl Serialize for LogEvent {
```

Comment:
Nice! This was on my list. :)

This is definitely outside of the scope of this PR though.